.vuepress/components/Home.vue

```diff
@@ -0,0 +1,314 @@
+    display: flex;
+    justify-content: flex-end;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+  }
+
+  &__icon {
+    width: 1.5rem;
+    height: 1.5rem;
+    fill: #aaa;
+    margin-right: 0.5rem;
+    transition: fill 0.15s ease-out;
+  }
+
+  &:hover &__icon {
+    fill: var(--color-text, black);
+  }
+}
+
+.intro {
+  width: 100%;
+  max-width: 800px;
+}
+
+.h1 {
+  font-size: 3rem;
+  font-weight: 700;
+  margin-bottom: 1.5rem;
+  line-height: 3.25rem;
+  letter-spacing: -0.02em;
+  padding-top: 2.5rem;
+}
+
+.h2 {
+  font-size: 2rem;
+  font-weight: 700;
+  margin-top: 4.5rem;
+  margin-bottom: 1rem;
+  line-height: 2.25rem;
+  letter-spacing: -0.01em;
+  padding: 1.5rem 0;
+}
+
+.p {
+  font-size: 1.5rem;
+  line-height: 2.25rem;
+
+  &__alt {
+    margin-top: 0.75rem;
+    margin-bottom: 2rem;
+    font-size: 1.25rem;
+    line-height: 1.75rem;
+  }
+}
+
+.sections {
+  display: grid;
+  margin-top: 3rem;
+  margin-bottom: 5rem;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  gap: 1.5rem;
+
+  &__item {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    min-height: 14.75rem;
+    position: relative;
+    color: initial;
+    border-radius: 0.5rem;
+    padding: 1.5rem;
+    background: linear-gradient(318.04deg, #FFFFFF 52.42%, #EBEDFF 81.42%);
+    box-shadow: 0px 2px 4px rgba(22, 25, 49, 0.05), 0px 0px 1px rgba(22, 25, 49, 0.2), 0px 0.5px 0px rgba(22, 25, 49, 0.05);
+    transition: box-shadow 0.25s ease-out, transform 0.25s ease-out, opacity 0.4s ease-out;
+
+    &:hover:not(:active) {
+      box-shadow: 0px 12px 24px rgba(22, 25, 49, 0.07), 0px 4px 8px rgba(22, 25, 49, 0.05), 0px 1px 0px rgba(22, 25, 49, 0.05);
+      transform: translateY(-2px);
+      transition-duration: 0.1s;
+    }
+
+    &:active {
+      transition-duration: 0s;
+      opacity: 0.7;
+    }
+
+    &__tag {
+      color: var(--tag-text-color);
+      font-size: 14px;
+      line-height: 20px;
+      letter-spacing: 0.03em;
+      text-transform: capitalize;
+      background: var(--tag-background-color);
+      border-radius: 4px;
+      padding: 2px 8px;
+      width: fit-content;
+    }
+
+    &__title {
+      font-weight: 500;
+      font-size: 1.25rem;
+      line-height: 1.75rem;
+      letter-spacing: -0.01em;
+      color: #000000;
+      margin-bottom: 0.5rem;
+    }
+
+    &__desc {
+      font-size: 0.875rem;
+      line-height: 1.25rem;
+      letter-spacing: 0.03em;
+      color: rgba(0, 0, 0, 0.667);
+    }
+  }
+}
+
+.stack {
+  display: grid;
+  gap: 1.5rem;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  margin-bottom: 4rem;
+
+  &__item {
+    color: var(--color-text, black);
+    transition: box-shadow 0.25s ease-out, transform 0.25s ease-out, opacity 0.4s ease-out;
+
+    &:active {
+      opacity: 0.7;
+      transition-duration: 0s;
+    }
+
+    &__image {
+      width: 100%;
+      object-fit: cover;
+    }
+
+    &__h1 {
+      font-size: 1.25rem;
```

Comment:
Let's reduce this size a bit. We can reduce it further when we increase to 4 columns - once we have another video to display.

```
font-size 1rem
line-height 1.5rem
margin-top 0.5rem
margin-bottom 0.25rem
```